src/predict_request_validation_utils.cpp

```diff
@@ -518,6 +631,24 @@ Status RequestValidator<KFSRequest, KFSTensorInputProto, KFSInputTensorIteratorT
     }
     return StatusCode::OK;
 }
+template <>
+Status RequestValidator<ovms::InferenceRequest, InferenceTensor, const InferenceTensor*, shape_t>::validateTensorContentSize(const InferenceTensor& proto, ovms::Precision expectedPrecision, size_t bufferId) const {
+    size_t expectedValueCount = 1;
+    for (size_t i = 0; i < proto.getShape().size(); i++) {
+        expectedValueCount *= proto.getShape()[i];
+    }
+    if (proto.getBuffer()->getByteSize()) {
```

Comment:
what happens when tensor claims that its buffer bytesize is 0?

Reply:
I assume that at this point we already validated that the InferenceTensor has Buffer attached to it?